src/main/java/org/myrobotlab/kinematics/Sequence.java

```diff
@@ -0,0 +1,45 @@
+package org.myrobotlab.kinematics;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.myrobotlab.codec.CodecUtils;
+import org.myrobotlab.io.FileIO;
+import org.myrobotlab.logging.LoggerFactory;
+import org.myrobotlab.service.InMoov2;
+import org.slf4j.Logger;
+
+/** represent a set of servo positions at a given point in time */
+public class Sequence {

```

Comment:
this sequence, really should be renamed to gesture, and it should be represented by a list of steps..
steps can be pose steps or speaking steps..  (perhaps others)